packages/mdc-tab/_mixins.scss

```diff
@@ -36,6 +37,17 @@
   }
 }
 
+@mixin mdc-tab-states-color($color) {
+  .mdc-tab__ripple {
+    @include mdc-states($color);
+  }
+}
+
+@mixin mdc-tab-ink-color($color) {
```

Comment:
Since this is here... should this also set mdc-tab-icon-color so that everything is set together? (I forget why text and icon color were originally separate, maybe @patrickrodee can weigh in on this.)

Reply:
We had separate mixins for text and icon in other components when I added these mixins. Setting `icon` color in the `ink` mixin makes sense to me.

Reply:
Makes sense, my definition of ink was outdated. Done.